spec/lib/lacerda/compare/json_schema_spec.rb

```diff
@@ -23,15 +29,16 @@
             'multi_props'=> { 'type'=>'array', 'items'=> [ {
               'oneOf' => [
                 { 'type' => 'string' },
-                { '$ref' => '#/definitions/post' } ]
+                { '$ref' => '#/definitions/date' } ]

```

Comment:
When I wrote this test, I didn't notice that this was a post that could reference another post, kind of recursively.  I've got some errors that were hard to track...so I just changed it for something a bit simpler.